src/config.c

```diff
@@ -396,7 +396,7 @@ struct SubnetList *parseSubnetAddress(char *addrstr) {
     }
 
     tmpSubnet = (struct SubnetList*) malloc(sizeof(struct SubnetList));
-    tmpSubnet->subnet_addr = addr;
+    tmpSubnet->subnet_addr = (addr & mask);
```

Comment:
Why is this change needed?

Reply:
Because in ifvc.c are now handling subnets on the form addr & mask so it must be changed in order to work properly.

Reply:
I just would like to note that this specific change was reverted on pfSense

https://github.com/pfsense/FreeBSD-ports/commit/009d60cab48f85b0e75a6a35bd7805ece19e6806

Reply:
You are right, yesterday night i had compiled the source code and test it. It is working perfectly without this change, and i can confirm that igmpproxy is working with a vlan tagged with pppoe interface.

What are next steps?. Do I cancel this pull request and open another one without config.c change or Do I revert the change in this pull request?

Reply:
Please update *this* pull request. There is no need to create a new one.